indra-common/src/main/kotlin/net/kyori/indra/IndraExtension.kt

```diff
@@ -117,6 +117,13 @@ open class IndraExtension @Inject constructor(objects: ObjectFactory) {
     url = "https://opensource.org/licenses/Apache-2.0"
   ))
 
+  fun gpl30OnlyLicense() = this.license.set(License(
```

Comment:
not a fan of the `30` part. should indicate the decimal somehow, or skip the `.0` -- the FSF seem to use `v3` alone in most places, with the `.0` only in their URL.